free-1.md

```diff
@@ -57,12 +70,16 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+This seems like something in which language design would actually be the core of the work. I imagine extracting and manipulating the frequencies from a sound file is something that would probably be borrowed from another language or package. This would leave the programmer (me) with more time to actually think about and implement language design that makes sense.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+I think this is actually totally doable, **provided** the idea of borrowing pitch bending/manipulation software or packages from elsewhere isn't ridiculous. I spend a short amount of time perusing the internet for just such a thing and couldn't find anything, so perhaps I am underestimating the difficulty of such a project. But at the very lest this is something I _do_ know how to do with FLStudio so I do feel something can be finagled. Also, this is similar to how autotune works, so I know this technology does exist in some form.
```

Comment:
I agree. As mentioned above, I foresee the biggest challenges being dealing with different codecs, which shouldn't be too bad.